### DISCUSSION TITLE
为什么B站上有的视频，版本1.5的解析不了，1.4的可以解析并下载？

### DISCUSSION THREAD

**Discussion by zbagogo** [leiurayer/downkyi#821] [Q&A]
一部歌剧的视频，UP主在2017年上传，视频长度为1小时35分钟。下载姬用1.5.7到1.5.9三个版本，复制链接后，只出现视频名称，但都解析不了。用1.4.1版本就能够解析、选择分辨率并下载。是因为UP主上传时间太早？还是什么其他原因？
Up主：Pesacarada、视频名称：2006年皇家歌剧院版《费加罗的婚礼》
视频链接：
https://www.bilibili.com/video/BV1fx411p7Ka/?spm_id_from=333.999.0.0&vd_source=ca51b2e9bc80b857d03ffc35d1a7726f

**Comment by fs1314** [comment]
还真是

**Comment by fs1314** [comment]
这个我今天也遇到了，BV1bx411s7u8，我用其他的解析也出不来，离谱，
